content/docs/cve/cveCreateSession.md

```diff
@@ -80,7 +83,7 @@ Detailed instructions to schedule an exam session in the ExamTools software are
 
 * Click on the icon and the link to your session on HamStudy will automatically copy to your clipboard.  You should be able to open a browser window and paste the url in and the address bar and hit enter.  Send the link to others or posted to another website to advertise your session.  
 
-* Search for your session by clicking on the **Find a Session** from the right side navigation on HamStudy.  If the session is not hidden, then it should come up.  Find all your sessions by adding your callsign to the end of the following link: https://ham.study/sessions/yourcallsign.  Use this url to link to all your sessions live on HamStudy.  Include it in the session descriptions or share in email. This link will not include hidden sessions.
+* Search for your session by clicking on the **Find a Session** from the right side navigation on [HamStudy](https://ham.study).  If the session is not hidden, then it should come up.  Find all your sessions by adding your callsign to the end of the following link: https://ham.study/sessions/yourcallsign.  Use this URL to link to all your sessions live on HamStudy.  Include it in the session description or share in email. This link will not include hidden sessions.
```

Comment:
> link: https://ham.study/sessions/yourcallsign.

Maybe format this with back-tics so it doesn't become clickable... Also, if says "add to the end of..." so perhaps leave out `yourcsallsign` or change the wording to say "replace `yourcallsign` with your callsign" or something along those lines.